CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [12.10.3]
 ### Fixed
 - fix: ensure recipient address is shown on when sending erc 1155 tokens ([#29775](https://github.com/MetaMask/metamask-extension/pull/29775))
```

Comment:
Two comments on this PR:

- the changelog should, ideally, link to the commit on main, not the cherry pick
- it should only include user facing changes